2013-12-23 14:01:48,030 INFO: microblog startup [in C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\__init__.py:33]
2013-12-23 14:01:56,099 ERROR: Exception on /edit [POST] [in c:\Python27\lib\site-packages\flask\app.py:1306]
Traceback (most recent call last):
  File "c:\Python27\lib\site-packages\flask\app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\Python27\lib\site-packages\flask\app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\Python27\lib\site-packages\flask\app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\Python27\lib\site-packages\flask\app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\Python27\lib\site-packages\flask_login.py", line 658, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\views.py", line 92, in edit
    if eForm.validate_on_submit():
  File "c:\Python27\lib\site-packages\flask_wtf\form.py", line 156, in validate_on_submit
    return self.is_submitted() and self.validate()
  File "C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\forms.py", line 28, in validate
    user = User.query.filter_by(nickname = self.nickname.data).first()
NameError: global name 'User' is not defined
2013-12-23 14:02:01,227 ERROR: Exception on /edit [POST] [in c:\Python27\lib\site-packages\flask\app.py:1306]
Traceback (most recent call last):
  File "c:\Python27\lib\site-packages\flask\app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\Python27\lib\site-packages\flask\app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\Python27\lib\site-packages\flask\app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\Python27\lib\site-packages\flask\app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\Python27\lib\site-packages\flask_login.py", line 658, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\views.py", line 92, in edit
    if eForm.validate_on_submit():
  File "c:\Python27\lib\site-packages\flask_wtf\form.py", line 156, in validate_on_submit
    return self.is_submitted() and self.validate()
  File "C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\forms.py", line 28, in validate
    user = User.query.filter_by(nickname = self.nickname.data).first()
NameError: global name 'User' is not defined
2013-12-23 14:02:11,203 INFO: microblog startup [in C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\__init__.py:33]
2013-12-23 14:02:11,599 INFO: microblog startup [in C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\__init__.py:33]
2013-12-23 14:03:08,125 INFO: microblog startup [in C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\__init__.py:33]
2013-12-23 14:03:15,864 INFO: microblog startup [in C:\Users\Shane\Documents\Coding\python\flask\trade_lister\app\__init__.py:33]
